.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "Code Scanning - Action"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Test
+        run: go test -v ./...
+
+security:
```

Comment:
Looks like this isn't properly formatted and the above action doesn't seem appropriate for this repository. Note how this check wasn't added as part of the checks for this PR.

Reply:
i will check that